libs/ui-lib/lib/ocm/components/clusterConfiguration/OcmClusterDetailsFormFields.tsx

```diff
@@ -220,10 +217,12 @@ export const OcmClusterDetailsFormFields = ({
         </StaticTextField>
       ) : (
         <ExternalPlatformDropdown
-          showOciOption={isOracleCloudPlatformIntegrationEnabled}
-          disabledOciTooltipContent={oracleDropdownItemState?.disabledReason}
-          isOciDisabled={oracleDropdownItemState?.isDisabled || false}
           onChange={handleExternalPartnerIntegrationsChange}
+          clusterPlatform={clusterPlatform}
+          clusterExists={clusterExists}
+          cpuArchitecture={cpuArchitecture ?? values.cpuArchitecture}
```

Comment:
I don't think `(architectureData[values.cpuArchitecture] as CpuArchitectureItem).label` and `values.cpuArchitecture` are interchangeable.

Pass `values.cpuArchitecture` as a prop here and then when you need the CPU architecture label (for a tooltip in the ExternalPlatformDropdown component), use the `architectureData` object there.

(I don't think you want to end up with a string like `"IBM Power (ppc64le)"` to get passed down all the way to the `getOciDisabledReason` function.)

Also, please update the prop types accordingly.

```suggestion
          cpuArchitecture={values.cpuArchitecture}
```